scripts/dist-core.js

```diff
@@ -17,6 +18,14 @@ async function build({ package: packageName, entry = 'src/index.js', name, shoul
   const bundle = await rollup.rollup({
     input: `./packages/${packageName}/${entry}`,
     plugins: [
+      packageName === 'rax' && format === 'iife' ? memory({
+        path: `./packages/${packageName}/${entry}`,
+        contents: `
+        import * as Rax from './index.js';
+        if (typeof module!='undefined') module.exports = Rax;
```

Comment:
typeof module!='undefined' -> typeof module != 'undefined'

Reply:
`!==` is better